include/IMU.hpp

```diff
@@ -10,4 +10,4 @@ class BOARD_NAME {
 private:
 };
 
-}
+}// namespace BOARD_NAME
```

Comment:
Make sure there is a newline at the end of this file

Reply:
Should be fixed in the latest commit